src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -399,6 +417,10 @@ private function createChoiceList(array $options)
         // Harden against NULL values (like in EntityType and ModelType)
         $choices = null !== $options['choices'] ? $options['choices'] : array();
 
+        if (null !== $options['choice_filter']) {
```

Comment:
`&& $choices`? Not sure it matters much... =/

Reply:
sure, should save some useless function calls :+1: